app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -671,6 +681,12 @@ class TabTrayView(
         private const val SELECTION_DELAY = 500
         private const val NORMAL_HANDLE_PERCENT_WIDTH = 0.1F
         private const val COLUMN_WIDTH_DP = 180
+        // UI specs say grid items need to have 16dp padding.
+        // The layout already has a 8dp padding.
+        // (also used as buffer space for the external border and outside translation of the play button)
+        // We leverage the double spacing issue to have 8dp * 2 as in-between items padding.
+        // Need to add another 8dp of padding to the parent.
```

Comment:
I think we can simplify this text a bit since we already have a pretty thorough explanation above. 

Something like:

> The remaining padding offset needed to provide a 16dp column spacing between the grid items.

Reply:
Sure. Thanks!